docs/Demo.md

```diff
@@ -134,49 +229,203 @@ curl $(oc get route -n pelorus deploytime-exporter -o=template='http://{{.spec.h
+- Create and resolve bugs in Github issues that exercise Pelorus metrics
+- View the changes to the `Mean Time to Restore` and `Change Failure Rate` metrics.
 
-The Pelorus and todolist application can be installed automatically.  Using the forked copy of [mig-demo-apps](https://github.com/konveyor/mig-demo-apps) referenced as `https://github.com/<your_org>/mig-demo-apps`, execute the following steps:
+
+### Pelorus configuration
+
+>**Note:**
+> * A users [Github personal access token](https://github.com/settings/tokens) is required
+> * The `PROJECTS` key's value is the fork of the mig-apps-demo repository.
+
+* Update the /var/tmp/values.yaml to match the following text. You may already have the deploytime and committime exporters configured from a previous example.
+
+        exporters:
+          instances:
+          - app_name: deploytime-exporter
+            exporter_type: deploytime
+            source_context_dir: exporters/
+            extraEnv:
+            - name: APP_FILE
+              value: deploytime/app.py
+            - name: LOG_LEVEL
+              value: DEBUG
+            - name: NAMESPACES
+              value: mongo-persistent
+            image_name: quay.io/pelorus/deploytime-exporter
+          - app_name: committime-exporter
+            exporter_type: committime
+            source_context_dir: exporters/
+            extraEnv:
+            - name: APP_FILE
+              value: committime/app.py
+            - name: LOG_LEVEL
+              value: DEBUG
+            - name: NAMESPACES
+              value: mongo-persistent
+            image_name: quay.io/pelorus/committime-exporter
+          - app_name: failure-exporter
+            source_context_dir: exporters/
+            exporter_type: failure
+            extraEnv:
+            - name: PROVIDER
+              value: github
+            - name: TOKEN
+              value: ghp_J<snip>
+            - name: PROJECTS
+              value: <your_org>/mig-demo-apps
+            image_name: quay.io/pelorus/failure-exporter
+
+* Documentation regarding values.yaml can be found [in the configuration section.](Configuration.md)
+  Apply the updated values for Pelorus by executing:
+
+    
+        helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+    
+
+>**Note:** Please pause to allow the failure exporter pod to build and deploy.
+
+* Check the output from the failure exporter.  No bugs should be found at this time.
+
+    
+        curl $(oc get route -n pelorus failure-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
+    
+
+### Github Issues
+
+Pelorus will utilize two tags to determine if a Github issue is associated with the todolist-mongo application.  We'll need the default `bug` tag. Additionally, Pelorus requires that all issues associated with a particular application be labelled with the `app.kubernetes.io/name=<app_name>` label.  This works the same way as the deployment configuration.
```

Comment:
Additionally, **by default** Pelorus requires that all issues associated with a particular application be labelled with the `app.kubernetes.io/name=<app_name>` label.  This works the same way as the deployment configuration.

Reply:
updated.. thanks